### PR TITLE
Get Auth config for web from backend

### PR DESCRIPTION
**Pull request by arthur-jx**

### CODE REVIEW

File: controllers/auth.go [hunk 1]
```diff
@@ -29,6 +30,21 @@ func init() {
 	auth.InitConfig(CasdoorEndpoint, ClientId, ClientSecret, JwtSecret, CasdoorOrganization)
 }
 
+func (c *ApiController) GetAuthConfig() {
+	app := beego.AppConfig.String("appname")
+	srv := beego.AppConfig.String("casdoorEndpoint")
```
Comment (kocoler) [inline controllers/auth.go]: `casdoorEndpoint` is different from `casdoor serverurl`, so there should be another field, `appname` is the same.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,12 @@ Backend | RESTful API backend for Casnode | Golang + Beego + MySQL | https://git
 
 
 - Setup database:
+    Create application config file from sample
```
Comment (kocoler) [inline README.md:41]: This configuration-related part seems more appropriate to write in `Custom settings` or `Run backend` part?

Comment (arthur-jx) [reply README.md:41]: Configuration comes from the same place, which can simplify project release and deployment

**Comment by hsluoyz** [comment]
@arthur-jx we can't call `/api/get-auth-config` API for each page view. It's costing so much.

**Comment by arthur-jx** [comment]
the config 

> @arthur-jx we can't call `/api/get-auth-config` API for each page view. It's costing so much.

authConfig is global object, Just get it once during WEB initialization. I think this is acceptable。

**Pull request closed without merging**